storage/kvstore/filesystemstore/tests/UNITTESTS/FileSystemStore/moduletest.cpp

```diff
@@ -73,6 +73,9 @@ TEST_F(FileSystemStoreModuleTest, erased_set_get)
     EXPECT_EQ(heap.init(), MBED_SUCCESS);
     EXPECT_EQ(heap.erase(0, heap.size()), MBED_SUCCESS);
     EXPECT_EQ(heap.deinit(), MBED_SUCCESS);
+    if (fs->mount(&heap) != MBED_SUCCESS) {
+        EXPECT_EQ(fs->reformat(&heap), MBED_SUCCESS);
+    }
```

Comment:
See https://github.com/ARMmbed/mbed-os/pull/12652#discussion_r605544000

Reply:
TODO: Remove this test case as per https://github.com/ARMmbed/mbed-os/pull/12652#discussion_r612145136